source/scss/variables/_color-variables.scss

```diff
@@ -5,36 +5,73 @@ $gray-400: #929497;
 $gray-600: #666666;
 $gray-800: #424242;
 
-$red: #fc1119;
+$dark-gray: $gray-800;
+$dark-gray-80: #676767;
+$dark-gray-60: #8c8c8c;
+$dark-gray-40: #b1b1b1;
+$dark-gray-20: #d6d6d6;
+
+$light-gray: $gray-100;
+$light-gray-80: #f1f1f1;
+$light-gray-60: #f3f3f3;
+$light-gray-40: #f6f6f6;
+$light-gray-20: #f8f8f8;
+
 $orange: #ff9933;
+$orange-80: #fead5b;
```

Comment:
We can define it as "primary" directly. Every design that is orange, will be names "primary" anyway. This could be seen as duplicated code.